webui/react/package.json

```diff
@@ -13,7 +13,7 @@
     "lint:js": "eslint \"src/**/*.{js,jsx,ts,tsx}\"",
     "lint:css": "stylelint --config .stylelintrc.js \"src/**/*.{css,less,scss}\"",
     "test": "craco test --config jest.config.js",
-    "test:coverage": "CI=true npm test -- --env=jsdom --coverage",
+    "test:coverage": "CI=true JEST_JUNIT_OUTPUT_DIR=coverage npm test -- --env=jsdom --coverage --reporters=default --reporters=jest-junit",
```

Comment:
this could be added to the jest-config too I think

Reply:
if that's easy to do, I think that's a good idea to do

Reply:
[related thread](https://stackoverflow.com/questions/59372493/override-jest-junit-default-output-location)

Reply:
https://github.com/determined-ai/determined/pull/3248/commits/70176e0780d5316b143a3c11e10ba469e5ae9840 this generates the correct config but seems to be overridden somewhere.

Reply:
**question:** idk much about the reporters. For these flags `--reporters=default --reporters=jest-junit` are they ok to have as part of `npm run test` since that also uses `jest.config.js`

Reply:
> 70176e0 this generates the correct config but seems to be overridden somewhere.

hmmm... there is a `jest` config in `package.json`:
```
  "jest": {
    "coveragePathIgnorePatterns": [
      "/src/services/api-ts-sdk/"
    ]
  }
```
could that be the culprit? should we move the package.json `jest` config into `jest.config.js` also?

Reply:
```
These options in your package.json Jest configuration are not currently supported by Create React App:

  • reporters

If you wish to override other Jest options, you need to eject from the default setup. You can do so by running npm run eject but remember that this is a one-way operation. You may also file an issue with Create React App to discuss supporting more options out of the box.
```

Reply:
reporters https://jestjs.io/docs/cli#--reporters instruct which reporters to run. we only want it when running in the ci since it's not more readable than the cli output in local usecase IMO

Reply:
> **question:** idk much about the reporters. For these flags `--reporters=default --reporters=jest-junit` are they ok to have as part of `npm run test` since that also uses `jest.config.js`

we'd only really want those options if we're looking at the junit report which AFAIK we'd only do through CI. adding it to `test` would just add additional work that `npm run test` would need to do if we run it locally

Reply:
I simplified the package.json scripts https://github.com/determined-ai/determined/pull/3248/commits/f4a6bf35284db1a788e45330358d691aca44cb77